docs/development/shareable-configs.md

```diff
@@ -50,12 +50,20 @@ If you use a non-scoped config, you must use a preset name!
 In general, GitHub, GitLab or Gitea-based preset hosting is easier than npm because you avoid the "publish" step - simply commit preset code to the default branch and it will be picked up by Renovate the next time it runs.
 An additional benefit of using source code hosting is that the same token/authentication can be reused by Renovate in case you want to make your config private.
 
-| name                    | example use          | preset    | resolves as                          | filename       |
-| ----------------------- | -------------------- | --------- | ------------------------------------ | -------------- |
-| GitHub default          | `github>abc/foo`     | `default` | `https://github.com/abc/foo`         | `default.json` |
-| GitHub with preset name | `github>abc/foo:xyz` | `xyz`     | `https://github.com/abc/foo`         | `xyz.json`     |
-| GitLab default          | `gitlab>abc/foo`     | `default` | `https://gitlab.com/abc/foo`         | `default.json` |
-| GitLab with preset name | `gitlab>abc/foo:xyz` | `xyz`     | `https://gitlab.com/abc/foo`         | `xyz.json`     |
-| Gitea default           | `gitea>abc/foo`      | `default` | `https://gitea.com/abc/foo`          | `default.json` |
-| Gitea with preset name  | `gitea>abc/foo:xyz`  | `xyz`     | `https://gitea.com/abc/foo`          | `xyz.json`     |
-| Local default           | `local>abc/foo`      | `default` | `https://github.company.com/abc/foo` | `default.json` |
+| name                               | example use                | preset    | resolves as                          | filename       | tag            |
```

Comment:
The tables in both the Markdown files are hard to read and review.
Could we fix this by making 4 tables, instead of having it all in one big table?

1. GitHub table
2. GitLab table
3. Gitea table
4. Local table